src/startup.rs

```diff
@@ -26,7 +27,7 @@ pub fn run(listener: TcpListener, settings: Settings) -> Result<Server, std::io:
             .route("/__version__", web::get().to(version))
             .route("/__heartbeat__", web::get().to(heartbeat))
             .route("/__lbheartbeat__", web::get().to(lbheartbeat))
-            .app_data(web::Data::new(symbol_manager.clone()))
+            .app_data(app_data.clone())
```

Comment:
When ever the app crashes, or it's scaling or something, this closure is called to construct an instance of the app. Since the data is basically just an `Arc` we can clone it here to avoid doing an allocation every time the app is recreated.